.travis.yml

```diff
@@ -11,5 +11,33 @@ cache:
   directories:
   - $HOME/.m2
 
-script:
-  - ./extras/travis_push_docker_hub.sh
+  
+stages:
+  - name: unit test
+    if: type IN (push, pull_request)
+  - name: integ test
+    if: type IN (push, pull_request)
+  - name: image build
+    if: type = push AND branch = master       
+    
+jobs:
+  include:
+    - stage: unit test
+      script: mvn test
+    - stage: integ test
+      before_script:
+        - sudo service postgresql stop
```

Comment:
What is this for? What Postgres are you stopping at this line?

Reply:
Disabling here PostgreSQL services enabled by default by Travis (https://docs.travis-ci.com/user/database-setup/#postgresql)

Reply:
I think PostgreSQL is preinstalled to Travis VMs, but not automatically launched. To launch it one needs to add it to `services` list: https://docs.travis-ci.com/user/database-setup/#postgresql So, I guess, you don't have to turn it off, because you didn't actually turn it on...